src/diffusers/utils/hub_utils.py

```diff
@@ -95,53 +94,43 @@ def http_user_agent(user_agent: Union[Dict, str, None] = None) -> str:
     return ua
 
 
-def create_model_card(args, model_name):
+def load_or_create_model_card(repo_id=None, token=None, is_pipeline=False) -> ModelCard:
+    """
+    Loads or creates a model card.
+
+    Args:
+        repo_id (`str`):
+            The repo_id where to look for the model card.
+        token (`str`, *optional*):
+            Authentication token. Will default to the stored token. See https://huggingface.co/settings/token for more details.
+        is_pipeline (`bool`, *optional*):
+            Boolean to indicate if we're adding tag to a [`DiffusionPipeline`].
+    """
     if not is_jinja_available():
         raise ValueError(
             "Modelcard rendering is based on Jinja templates."
             " Please make sure to have `jinja` installed before using `create_model_card`."
             " To install it, please run `pip install Jinja2`."
         )
 
-    if hasattr(args, "local_rank") and args.local_rank not in [-1, 0]:
```

Comment:
Follow the discussion here: https://github.com/huggingface/diffusers/pull/6678#discussion_r1463056454.